src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.LocalFunctions.cs

```diff
@@ -133,6 +134,27 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
 
```

Comment:
The parameter default values should be visited with a temporarily set *unreachable* state to suppress definite assignment errors. Since these are required to be constant, any definite assignment errors would just be cascaded diagnostics.